ethdb/remote/ethbackend.pb.go

```diff
@@ -1,459 +1,364 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
-// versions:
-// 	protoc-gen-go v1.25.0
-// 	protoc        v3.13.0
 // source: remote/ethbackend.proto
 
 package remote
 
 import (
+	fmt "fmt"
 	proto "github.com/golang/protobuf/proto"
-	protoreflect "google.golang.org/protobuf/reflect/protoreflect"
-	protoimpl "google.golang.org/protobuf/runtime/protoimpl"
-	reflect "reflect"
-	sync "sync"
+	math "math"
 )
 
-const (
-	// Verify that this generated code is sufficiently up-to-date.
-	_ = protoimpl.EnforceVersion(20 - protoimpl.MinVersion)
-	// Verify that runtime/protoimpl is sufficiently up-to-date.
-	_ = protoimpl.EnforceVersion(protoimpl.MaxVersion - 20)
-)
+// Reference imports to suppress errors if they are not otherwise used.
+var _ = proto.Marshal
+var _ = fmt.Errorf
+var _ = math.Inf
 
-// This is a compile-time assertion that a sufficiently up-to-date version
-// of the legacy proto package is being used.
-const _ = proto.ProtoPackageIsVersion4
+// This is a compile-time assertion to ensure that this generated file
+// is compatible with the proto package it is being compiled against.
+// A compilation error at this line likely means your copy of the
+// proto package needs to be updated.
+const _ = proto.ProtoPackageIsVersion3 // please upgrade the proto package
```

Comment:
Is this a downgrade?

Reply:
It depends on what generator version installed locally. 

But i found solution for this: if put exact version to go.mod and install generator by ‘go install’ from app folder (make devtools - does it now) - then it installs version from go.mod file. 

So, if run make devtools in current master and re-gen things, then ProtoPackageIsVersion6 must appear

Reply:
Hmm... I've cleaned module packages and downloaded them once again, and now it generates "SupportPackageIsVersion7". At the end of the day, I updated grpc package and regenerated all files once again. It seems everything is good now and tests are green